compiler/luci-interpreter/src/KernelBuilder.cpp

```diff
@@ -166,6 +167,20 @@ std::unique_ptr<Kernel> KernelBuilder::visit(const luci::CircleMaxPool2D *node)
   return std::make_unique<kernels::MaxPool2D>(input, output, params);
 }
 
+std::unique_ptr<Kernel> KernelBuilder::visit(const luci::CircleMean *node)
+{
+  assert(node->arity() == 2);
```

Comment:
One last note (sorry for being late, I've just noticed):
Please throw an exception if `node->reduction_indices()` is not a constant. You can refer to `Reshape` and `Pad` for an example.

Reply:
Done. Thanks.